@types/omit-empty-es/index.d.ts

```diff
@@ -1,4 +0,0 @@
-declare module 'omit-empty-es' {
```

Comment:
This is not needed anymore to how I understand it. The issue with having it is that TypeScript prefers the `@types` over any bundled type of the `omit-empty-es`-library. It shadows the actual types which have changed and uses these outdated types which do not support a generic and resolve to `Record<string, unknown>` which we can't assign to anything.

Reply:
Yes thanks